app/helpers/candidates/school_helper.rb

```diff
@@ -129,7 +129,7 @@ def content_or_msg(content, msg = nil)
       msg = content_tag(:em, msg)
     end
 
-    content.presence || msg
+    content.present? ? sanitize(content) : msg
```

Comment:
This is my only question - why is sanitize getting called here?

this will strip most but not all tags from `content`, and mark the resultant string `html_safe`

For a consumer of this API that means its forced to accept some HTML tags, where previously it would just get a 'non html-safe' string which it could intepret as it wanted - the default being to escape html content.

Reply:
Ok - after some discussion - this is because HTML is getting passed through this helper - I don't think that should be happening anyway since a blank string will never be passed through. `_phase2.html:16` needs to stop using this helper method (particularly since it'll never do anything)

Add a helper to take the line orientated output and format it as bullets

And change the DB to recording line separated items for requirements?